### ISSUE TITLE
False positive for a test example

### ISSUE THREAD

**Issue by zeynepsu** [hhatto/nude.py#10]
I cloned the project and ran the tests immediately and got "True" for damita2.jpg from the examples. I will investigate to see what causes this.

**Comment by zeynepsu** [comment]
I confirmed that it is the ycbcr classifier that was added to this port of nude.js that causes this discrepancy between nude.js and nude.py. I will be doing more tests to see which one is beneficial to my specific use case.

**Comment by gcosta87** [comment]
Hi!
I recently discovered this project, looking for software that detects nudity, to analyze a large number of photos of user profiles. I do not program in Python (yet).

I wanted to report the same problem thas was reported by zeynepsu, and even based on her comment I tried disabling the «ycbcr_classifier» in th line *nude.py:312*, and got better results on a set of 40 images of example, where 20 have nudity and other 20 without it.

In short, to analyze the images and enabling / disabling that classifier, I got the following misclassified images:

image type  | Enabled | Disabled
------ | ---- | ------
nude  | 11 | 6
no nude | 8 | 5

Note: The test images were taken from this very small script witen in Python too: [pornscanner](http://www.warriorhut.org/graphics/pornscanner/pornscanner.zip)
